ledger/accountdb.go

```diff
@@ -1751,6 +1780,71 @@ func performResourceTableMigration(ctx context.Context, tx *sql.Tx, log func(pro
 	return nil
 }
 
+func performOnlineAccountsTableMigration(ctx context.Context, tx *sql.Tx, log func(processed, total uint64)) (err error) {
+	var insertNewAcctBase *sql.Stmt
+	var insertNewAcctBaseNormBal *sql.Stmt
+	insertNewAcctBase, err = tx.PrepareContext(ctx, "INSERT INTO onlineaccounts(address, data) VALUES(?, ?)")
+	if err != nil {
+		return err
+	}
+	defer insertNewAcctBase.Close()
+
+	insertNewAcctBaseNormBal, err = tx.PrepareContext(ctx, "INSERT INTO onlineaccounts(address, data, normalizedonlinebalance) VALUES(?, ?, ?)")
+	if err != nil {
+		return err
+	}
+	defer insertNewAcctBaseNormBal.Close()
+
+	var rows *sql.Rows
+	rows, err = tx.QueryContext(ctx, "SELECT address, data, normalizedonlinebalance FROM accountbase ORDER BY address")
+	if err != nil {
+		return err
+	}
+	defer rows.Close()
+
+	var insertRes sql.Result
+	var rowsAffected int64
+	var processedAccounts uint64
+	var totalBaseAccounts uint64
+
+	totalBaseAccounts, err = totalAccounts(ctx, tx)
+	if err != nil {
+		return err
+	}
+	for rows.Next() {
```

Comment:
this loop isn't "just right". You need to do the following:
for each account data:
   if it's not an online account ( determined by Status ), ignore it.
   otherwise:
      extract the voting data into the online account table
      remove the voting data from the acctbase ( please do that using row id )
      <later on> save the account data before and after so that we can compute the hash and update the merkle trie.

Reply:
if you want this PR to be "temporary", then I guess it's fine to do the above.

Reply:
yes, this is temporal to unblock parallel work on au/ct and ao trackers.